### PR TITLE
Remove permutation test score

### PR DESCRIPTION
**Pull request by ogrisel**
Address #449 to avoid using `permutation_test_score` to define chance level as it was found to be confusing and it's arguably just one way to define a chance level, not "the" way.

The quizz is being adapted accordingly in the private repo:

https://gitlab.inria.fr/learninglab/mooc-scikit-learn/mooc-scikit-learn-coordination/-/merge_requests/32

The exercise instructions and the quizz instructions will be synced in a follow-up commit once those 2 PRs are reviewed and merged.

### CODE REVIEW

File: python_scripts/cross_validation_sol_02.py [hunk 1]
```diff
@@ -36,97 +37,127 @@
...
+test_score_logistic_regression
 
 # %% [markdown]
-# Finally, compute the test score of a dummy classifier which would predict
-# the most frequent class from the training set. You can look at the
-# `sklearn.dummy.DummyClassifier` class.
+# Now, compute the cross-validation scores of a dummy classifier that
+# constantly predicts the most frequent class observed the training set. Please
+# refer to the online documentation for the [sklearn.dummy.DummyClassifier
+# ](https://scikit-learn.org/stable/modules/generated/sklearn.dummy.DummyClassifier.html)
```
Comment (lesteve) [inline python_scripts/cross_validation_sol_02.py:60]: Weird rendering here:
image

File: python_scripts/cross_validation_baseline.py [hunk 2]
```diff
@@ -1,23 +1,18 @@
 # %% [markdown]
 # # Comparing results with baseline and chance level
```
Comment (lesteve) [inline python_scripts/cross_validation_baseline.py:2]: The title needs to be changed "Comparing results with baseline" or something similar since we don't mention chance level any more.

**Review by lesteve — COMMENTED** [review]
Overall looks good to me, two minor comments.

File: python_scripts/cross_validation_baseline.py (hunk 2 above)
Comment (ogrisel) [reply python_scripts/cross_validation_baseline.py:2]: Good points. Thanks for the review, I will push some fixes.

**Comment by ogrisel** [comment]
Thanks for the review. The rendering looks good now. Once merged we will need to push another commit to update the exercise. Same for the private quiz solutions.

**Comment by lesteve** [comment]
Merging, thanks!

I added a "FUN: requires changes" label since this changes a notebook title
